perception/crosswalk_traffic_light_estimator/src/node.cpp

```diff
@@ -237,7 +237,7 @@ void CrosswalkTrafficLightEstimatorNode::setCrosswalkTrafficSignal(
       output_traffic_light.color = color;
       output_traffic_light.confidence = 1.0;
       output_traffic_signal.elements.push_back(output_traffic_light);
-      output_traffic_signal.traffic_light_id = ll_traffic_light.id();
+      output_traffic_signal.traffic_signal_id = ll_traffic_light.id();
```

Comment:
`traffic_signal_id` should represent the regulatory element id of the traffic light.
Therefore this node needs to do the [groupFusion](https://github.com/tier4/autoware.universe/blob/ed5fdace2cc26dd1c353c9daf6aa738ca4f6293c/perception/traffic_light_multi_camera_fusion/src/node.cpp#L292-L321), which is currently performed by `traffic_light_multi_camera_fusion`.

Current Message Output:
```
signals:
- traffic_signal_id: 144907
  elements:
  - color: 0
    shape: 0
    status: 0
    confidence: 1.0
- traffic_signal_id: 144918
  elements:
  - color: 0
    shape: 0
    status: 0
    confidence: 1.0
```

Expected Message Output:
(Note: ID 144912 represents the regulatory element ID which associates with the traffic light IDs 144907 and 144918.)
```
- traffic_signal_id: 144912
  elements:
  - color: 0
    shape: 0
    status: 0
    confidence: 1.0
```

Reply:
Definitely! Thank you for your comment. I apologize for my mistake.

Reply:
@TomohitoAndo Fixed at b5458c59e18519e71b547068e0fb8dde27bc3a3a